tests/unit/ssg-module/test_build_cpe.py

```diff
@@ -90,3 +92,49 @@ def test_extract_referred_nodes():
 
     assert len(results) == 1
     assert results[0].text == 'Source Code'
+
+
+def test_product_cpes():
+    product_yaml_path = os.path.join(DATADIR, "product.yml")
+    product_yaml = open_raw(product_yaml_path)
+    product_yaml["product_dir"] = os.path.dirname(product_yaml_path)
+    product_cpes = ssg.build_cpe.ProductCPEs(product_yaml)
+
+    # get CPE by name and verify it's loaded
+    rhel7_cpe = product_cpes.get_cpe("rhel7")
+    assert(rhel7_cpe.name == "cpe:/o:redhat:enterprise_linux:7")
+    assert(rhel7_cpe.title == "Red Hat Enterprise Linux 7")
+    assert(rhel7_cpe.check_id == "installed_OS_is_rhel7")
+    assert(rhel7_cpe.bash_conditional == "")
+    assert(rhel7_cpe.ansible_conditional == "")
+
+    # get CPE by ID and verify it's loaded
+    rhel7_cpe_2 = product_cpes.get_cpe("cpe:/o:redhat:enterprise_linux:7")
+    assert(rhel7_cpe_2.name == "cpe:/o:redhat:enterprise_linux:7")
```

Comment:
>>    or to at least compare cpe2 attribute to cpe1 attributes and not to a literal value.
>
> What do you mean by that?

```suggestion
    assert(rhel7_cpe_2.name == rhel7_cpe.name)
```

and so on - this strongly indicates that those returned CPE classes should be the same, and there is less copy-pasting of text.